src/components/TrackBulkActions/TrackBulkActions.tsx

```diff
@@ -52,20 +55,19 @@ const TrackBulkActions: React.FC<Props> = ({
       {selectionMode && (
         <div className={styles.controls}>
           <span data-testid="selection-count">
-            Selected: {selectedCount} / {totalCount}
+            Selected: {selectedTracks.length} / {totalCount}
           </span>
           <button
-            onClick={onSelectAll}
+            onClick={() => handleSelectAll(allTracks.map((t) => t.id))}
```

Comment:
To much going on in onClick,I suggest moving this method creation inside component body.